boilerplate/_lib/container-make

```diff
@@ -14,13 +14,13 @@ CONTAINER_ENGINE=$(command -v podman || command -v docker)
 
 # First set up a detached container with the repo mounted.
 banner "Starting the container"
-container_id=$($CONTAINER_ENGINE run -d -v "$REPO_ROOT":"$REPO_ROOT" $IMAGE_PULL_PATH tail -f /dev/null)
+container_id=$($CONTAINER_ENGINE run -d -v "$REPO_ROOT":"/go/src/github.com/openshift/aws-account-operator" $IMAGE_PULL_PATH tail -f /dev/null)
```

Comment:
Any reason for moving away from the $REPO_ROOT variable here?

Reply:
Good catch, this actually shouldn't be modified here.